### DISCUSSION TITLE
Hacktoberfest 2022 🎉

### DISCUSSION THREAD

**Discussion by turolopezsanabria** [SUI-Components/sui-components#2313] [Announcement]
# Adevinta's Hacktoberfest 2022 @SUI-Components Org

<a href="https://hacktoberfest.com/">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/23620759/191723103-08a0ed20-3252-44d7-b050-6a0f35338bfd.png">
    <img width="2400" alt="Email Banners-Dark. Light: 'Light' Dark: 'Dark'" src="https://user-images.githubusercontent.com/1674036/192119490-de23bc33-b42d-46fc-aeb0-eca55a3d65fd.png">
  </picture>
</a>

Hacktoberfest is a @DigitalOcean annual event that encourages people to contribute to [open-source](https://github.com/open-source) throughout October. Much of modern tech infrastructure including some [@Adevinta](https://www.adevinta.com/) own products rely on open-source projects built and maintained by passionate people who often don't have staff or budgets to do much more than keep the project alive. Hacktoberfest is all about giving back to those projects, sharpening skills, and celebrating all things open source, especially the people that make open source special.

For the last 9 years, thousands of people-coders and non-coders alike have participated in Hacktoberfest to support the projects they use and love, learn and practice skills that will enhance their careers, and meet new people who love open source as much as they do.

https://hacktoberfest.com/

# PARTICIPATION

### Everyone is welcome
> Participants in Hacktoberfest come from all over the world and represent thousands of unique skill sets. We welcome everyone who’s already part of the open-source software community and anyone who is interested in diving in.

### Quantity is fun, Quality is key
> Participating in Hacktoberfest leads to personal growth, professional opportunities, and community building. But it all begins with meaningful contributions to open-source software. We’re committed to prioritizing quality contributions as outlined in our participation rules.

### Short-term action, Long-term impact
> In the open-source community, we stand on the shoulders of project maintainers and those who came before us. Your participation has a lasting effect on people and technology long after October. Your participation helps build the future of the internet.

<img width="145px" alt="brand-2020" src="https://user-images.githubusercontent.com/1674036/192470254-73762667-ecf3-4bd2-97cb-63501916f323.png" /><img width="145px" alt="brand-2019" src="https://user-images.githubusercontent.com/1674036/192470273-2e50adbc-2355-42fd-ab67-5e7dbdc2f17c.png" /><img width="145px" alt="brand-2018" src="https://user-images.githubusercontent.com/1674036/192470286-76f67f19-4acb-4248-8a1c-012af8457be0.png" /><img width="145px" alt="brand-2017" src="https://user-images.githubusercontent.com/1674036/192470319-d43b1491-e3f7-4d20-8361-f6a2c50909d8.png" /><img width="145px" alt="brand-2016" src="https://user-images.githubusercontent.com/1674036/192470337-1d87fb0c-e157-4c4d-99b3-73b70c78354a.png" /><img width="145px" alt="brand-2015" src="https://user-images.githubusercontent.com/1674036/192470351-b42a0976-85e7-41f0-954b-ca3a8233e034.png" />

# CONTRIBUTORS

HERE’S WHAT YOU NEED TO KNOW TO PARTICIPATE AND COMPLETE HACKTOBERFEST:

- Register anytime between September 26 and October 31
- Pull requests can be made in any [GITHUB](https://github.com/topics/hacktoberfest) or [GITLAB](https://gitlab.com/explore/projects/topics/hacktoberfest) hosted project that’s participating in Hacktoberfest (look for the “hacktoberfest” topic)
- Project maintainers must accept your pull/merge requests for them to count toward your total
- Have 4 pull/merge requests accepted between October 1 and October 31 to complete Hacktoberfest

The first 40,000 participants (maintainers and contributors) who complete Hacktoberfest can elect to receive one of two prizes: a tree planted in their name, or the Hacktoberfest 2022 t-shirt. You can find [here](https://hacktoberfest.com/participation/#beginner-resources) many resources for beginners.

# LOW OR NON CODE CONTRIBUTIONS
Contributing to open source isn't just for technical folks who want to write code. There are many opportunities to use your professional skills in support of open-source projects. This year, @digitalocean is making a point to encourage contributions that require some technical experience or none at all. No matter your experience, you can participate in Hacktoberfest!

Hacktoberfest welcomes people of any experience level to participate, and low-code and no-code contributions are fantastic choices for folks with little technical knowledge. Here are some examples of ways you can contribute to open-source projects.

|          | LOW-CODE                                                                           | NON-CODE                                                            |
|:----------:|------------------------------------------------------------------------------------|---------------------------------------------------------------------|
|  📝<br>WRITING | - Technical documentation                                                          | - Translating<br>- Copy Editing                                     |
|  👨‍🎨<br>DESIGN | - Testing                                                                          | - User Experience Testing<br>- Graphic design<br>- Video Production |
| 🎓<br>ADVOCACY | - Talks or presentations<br>- Technical blog posts<br>- Podcasts<br>- Case studies | - Social media<br>- Blog posts                                      |

*Contributors submitting low/non-code content to projects SHOULD create a PR/MR to track it. While Hacktoberfest tracks all PR/MRs submitted for the event, Maintainers may need to facilitate tracking of those contributions through an activity log or similar.*

<p align="center">
  <img height="150" alt="stickers" src="https://user-images.githubusercontent.com/1674036/192471662-e42cd3bb-fd53-4d6b-98ca-97c04e9e7f97.png">
</p>

---
[2021 Edition](https://github.com/SUI-Components/sui-components/discussions/1778)

**Comment by andresin87** [comment]
# [Table Board](https://github.com/orgs/SUI-Components/projects/1/views/4)

The Adevinta Spain sui-components maintainers Team had prepared a board with some proposed tasks we consider interesting to help us reducing bugs and legacy, some refactors and increasing our coverage specially.
